doc/content/verification_and_validation/ver-1ia.md

```diff
@@ -0,0 +1,69 @@
+# ver-1ia
+
+# Species Equilibration Problem in Ratedep Conditions with Equal Starting Pressures
+
+## General Case Description
+
+This verification problem is taken from [!cite](ambrosek2008verification). When two species can react on a surface to form a third, it is possible to predict the rate at which equilibration between the species will occur. For example, the reaction between two isotopic species, A$_2$ and B$_2$, is described as
+
+\begin{equation}
+\label{eq:reaction}
+\frac{1}{2} A_2 + \frac{1}{2} B_2 \rightleftharpoons AB,
+\end{equation}
+
+and the partial pressure of A$_2$, B$_2$, and AB in equilibrium of the reaction is defined by
+
+\begin{equation}
+\label{eq:reaction_equilibrium_pressure}
+\frac{P_{AB}}{\sqrt{P_{A_2}} \sqrt{P_{B_2}}} = K_{eq},
+\end{equation}
+
+where $P_i$ is the partial pressure of corresponding gas $i$, $K_{eq}$ is the equilibrium constant, and $K_{eq} = 2$ in this isotope reaction. Therefore, the partial pressure of AB in equilibrium is a constant value depends on initial partial pressure of A$_2$ and B$_2$:
+
+\begin{equation}
+\label{eq:p_AB_equilibrium}
+P_{AB}^{eq} = \frac{2 P_{A_2}^0 P_{B_2}^0}{P_{A_2}^0 + P_{B_2}^0}.
+\end{equation}
+
+Under ratedep condition, the conversion rate at the surface is higher than the rate in enclosure. We solve
```

Comment:
Please use the information from Appendix A in [!cite](ambrosek2008verification) to better explain what ratedep is. 
The same comment applies to the other approximations in the other cases. We need the documentation here to explain what these conditions are.